### PR TITLE
Enhance/disk setup

### PR DESCRIPTION
**Pull request by shahnawaz-creator**

### CODE REVIEW

File: chimney.sh [hunk 1]
```diff
@@ -28,6 +28,22 @@ export BLOBBER_WALLET_PRIV_KEY=0chainblobwalletprivkey
 
 export DEBIAN_FRONTEND=noninteractive
 
+export PROJECT_ROOT_SSD=/var/0chain/blobber/ssd
+export PROJECT_ROOT_HDD=/var/0chain/blobber/hdd
+
+mkdir -p $PWD/disk-setup/
+wget https://raw.githubusercontent.com/0chain/zcnwebappscripts/enhance/disk-setup/disk-setup/disk_setup.sh -O $PWD/disk-setup/disk_setup.sh
```
Comment (boddumanohar) [inline chimney.sh]: we might have to move this after line 48 where we do `apt install`

File: chimney.sh [hunk 2]
```diff
@@ -28,6 +28,22 @@ export BLOBBER_WALLET_PRIV_KEY=0chainblobwalletprivkey
...
+wget https://raw.githubusercontent.com/0chain/zcnwebappscripts/enhance/disk-setup/disk-setup/disk_func.sh -O $PWD/disk-setup/disk_func.sh
+
+sudo chmod +x $PWD/disk-setup/disk_setup.sh
+bash $PWD/disk-setup/disk_setup.sh $PROJECT_ROOT_SSD $PROJECT_ROOT_HDD
+
+## cleanup server before starting the deployment
+docker-compose -f /var/0chain/blobber/docker-compose.yml down --volumes || true
+docker-compose -f /var/0chain/blobber/zchain-compose.yml down --volumes || true
```
Comment (boddumanohar) [inline chimney.sh]: we will have to run all the docker-compose commands after we install docker-compose. 

Also now that we passing blobber and validator keys as a part of the script. This compose is not required `blobber/zchain-compose.yml`

**Review by boddumanohar — CHANGES_REQUESTED** [review]

File: chimney.sh (hunk 1 above)
Comment (shahnawaz-creator) [reply chimney.sh]: done

File: chimney.sh (hunk 2 above)
Comment (shahnawaz-creator) [reply chimney.sh]: done

**Comment by shahnawaz-creator** [comment]
@boddumanohar 
can you please review it again now.

**Review by boddumanohar — APPROVED** [review]
LGTM

**Pull request merged**